vendor/github.com/keybase/client/go/protocol/keybase1/simple_fs.go

```diff
@@ -548,6 +552,8 @@ type SimpleFSInterface interface {
 	// Close OpID, cancels any pending operation.
 	// Must be called after list/copy/remove
```

Comment:
I don't understand why separate close and cancel should be required, but the comments on Close should at least be changed. The "must be called after" part I don't think is true any more. @taruti ? 

Also, Taru said we should make the size argument a long instead of int for simpleFSRead and ReadArgs - can we get thjis in too?

In case the .avdl change was in a different PR: never mind!